packages/key-management/src/util/stubSignTransaction.ts

```diff
@@ -9,19 +9,31 @@ import uniqWith from 'lodash/uniqWith';
 const randomHexChar = () => Math.floor(Math.random() * 16).toString(16);
 const randomPublicKey = () => Crypto.Ed25519PublicKeyHex(Array.from({ length: 64 }).map(randomHexChar).join(''));
 
-export const stubSignTransaction = async (
-  txBody: Cardano.TxBody,
-  knownAddresses: GroupedAddress[],
-  inputResolver: Cardano.InputResolver,
-  extraSigners?: TransactionSigner[],
-  { additionalKeyPaths = [] }: SignTransactionOptions = {}
-): Promise<Cardano.Signatures> => {
+export interface StubSignTransactionProps {
+  txBody: Cardano.TxBody;
+  knownAddresses: GroupedAddress[];
+  inputResolver: Cardano.InputResolver;
+  extraSigners?: TransactionSigner[];
+  dRepPublicKey: Crypto.Ed25519PublicKeyHex;
+  signTransactionOptions?: SignTransactionOptions;
+}
+// { additionalKeyPaths = [] }: SignTransactionOptions = {}
```

Comment:
Is this trash?

Reply:
done: [a1b9746](https://github.com/input-output-hk/cardano-js-sdk/pull/919/commits/a1b9746d3ba8ca2f2e9c5bbd298ba538c1848364)